recipes/laravel-horizon/README.md

```diff
@@ -0,0 +1,32 @@
+# Laravel: Horizon
+
+This is an step-by-step guide to install [Laravel Horizon](https://laravel.com/docs/master/horizon) into your project.
+This expects the project to already have an working laravel project.
+
+## Steps
+
+1. Install redis using
+   the [service example](https://github.com/drud/ddev-contrib/blob/master/docker-compose-services/redis)
+2. Configure the redis configuration in `.env`
+3. Run `ddev composer require laravel/horizon`
+4. Run `ddev artisan horizon:install`
+5. Run horizon process in DDEV Installation
+
+    * Option A: Run using nohup
+
+      Use the `dot.ddev/config.laravel-horizon.yaml`
```

Comment:
Please add a little more detail here. This is great stuff, but people may not understand it. 
```suggestion
      Copy the `dot.ddev/config.laravel-horizon.yaml` into your project's .ddev directory, or incorporate it into your .ddev/config.yaml. This snipped uses a post-start hook to use artisan to start the horizon process. 
```